tools/scripts/generate_binary_build_matrix.py

```diff
@@ -430,12 +430,15 @@ def main(args) -> None:
         choices=[ENABLE, DISABLE],
         default=os.getenv("WITH_CUDA", ENABLE),
     )
+    # By default this is false for this script but expectation is that the caller
+    # workflow will default this to be true most of the time, where a pull
+    # request is synchronized and does not contain the label "ciflow/binaries/all"
     parser.add_argument(
-        "--limit-win-builds",
-        help="Limit windows builds to single python/cuda config",
+        "--limit-pr-builds",
```

Comment:
Is this specific to PR testing? How about release testing, this field would be false by default? 
Should it just be --limit-builds or --limit-python-builds?